src/components/cron.ts

```diff
@@ -126,3 +130,27 @@ export const createCoffeeChatCron = (client: Client): CronJob =>
       throw 'Bad channel type';
     }
   });
+
+export const assignCodeyRoleForLeaderboard = (client: Client): CronJob =>
+  new CronJob('0 0 0 */1 * * *', async function () {
+    const leaderboard = await getCoinLeaderboard(10);
```

Comment:
10 here is a "magic number" - we should put this as a constant at the top of the file (say "numberUsersToAssignRole") and use that instead.